proto/lavanet/lava/pairing/query.proto

```diff
@@ -120,7 +120,6 @@ message QueryProvidersRequest {
 
 message QueryProvidersResponse {
   repeated lavanet.lava.epochstorage.StakeEntry stakeEntry = 1 [(gogoproto.nullable) = false];
-           string                               output     = 2;
```

Comment:
when removing a field in proto you need to mark it as reservec

Reply:
Fixed in 94cd4b252152f7b27685976c085fb4e0fcdc02d0